### PR TITLE
Pull changes from https://github.com/ethereum/yellowpaper/pull/151/fi…

### PR DESCRIPTION
**Pull request by jamesray1**
…https://github.com/ethereum/yellowpaper/pull/151/files#diff-9f702e1491c55da9d76a68d651278764R919

### CODE REVIEW

**Comment by pirapira** [comment]
It doesn't look better on PDF.

**Comment by jamesray1** [comment]
Please reopen this, I made a change on the branch to fix the presentation. It is not just to fix the presentation.

**Comment by pirapira** [comment]
What is the purpose of this PR?

**Comment by jamesray1** [comment]
It's to pull in the changes from #151.

File: Paper.tex [hunk 1]
```diff
@@ -959,24 +959,27 @@ \subsubsection{Machine State}
...
 The exceptional halting function $Z$ is defined as:
-\begin{equation}
+\begin{equation} 
 Z(\boldsymbol{\sigma}, \boldsymbol{\mu}, I) \equiv
 \begin{array}[t]{l}
-\boldsymbol{\mu}_{\mathrm{g}} < C(\boldsymbol{\sigma}, \boldsymbol{\mu}, I) \quad \vee \\
-\mathbf{\delta}_{\mathrm{w}} = \varnothing \quad \vee \\
-\lVert\boldsymbol{\mu}_{\mathbf{s}}\rVert < \mathbf{\delta}_{\mathrm{w}} \quad \vee \\
-( w \in \{ \text{\small JUMP}, \text{\small JUMPI} \} \quad \wedge \\ \quad \boldsymbol{\mu}_{\mathbf{s}}[0] \notin D(I_{\mathbf{b}}) ) \quad \vee \\
+\boldsymbol{\mu}_g < C(\boldsymbol{\sigma}, \boldsymbol{\mu}, I) \quad \vee \\
```
Comment (pirapira) [inline Paper.tex]: I think this `g` should be still roman.

**Pull request closed without merging**